Gopkg.toml

```diff
@@ -0,0 +1,84 @@
+ignored = ["github.com/kubeflow/xgboost"]
```

Comment:
Do you need all these dependencies here yet? I'd suggest we add them when needed.

Reply:
I'll remove unused dep.